components/LevitatingLink.js

```diff
@@ -1,7 +1,7 @@
 export default function LevitatingLink ({ href, children }) {
   return (
-    <a href={href}>
-      <span className='text'>{children}</span>
+    <a href={href} >
+      <span>{children}</span>
```

Comment:
Please revert the changes in this file because it is breaking the link animation. (The text does not levitate when hover)